### PR TITLE
Image service task with gRPC and optional k8s 

### PR DESCRIPTION
**Pull request by kuceriklukas**

### CODE REVIEW

File: image-service-grpc/README.md [hunk 1]
```diff
@@ -2,62 +2,60 @@
...
-- How would you horizontally scale your service? 
 - If we wanted to store the images for further processing - how and where would you store them?
 - What would you change to support rasterization of PDFs?
+- What do you think are the advantages (and disadvantages) of using gRPC for inter-service communication over regular HTTP communication?
+
+## Optional: Trying out the code in kubernetes with Minikube
+
+### What you'll need
+- [Docker](https://docs.docker.com/engine/install/) (optional)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start) (optional)
```
Comment (michalderdak) [inline image-service-grpc/README.md:49]:
```suggestion
- [Docker](https://docs.docker.com/engine/install/)
- [Minikube](https://minikube.sigs.k8s.io/docs/start)
```

Comment (michalderdak) [reply image-service-grpc/README.md:49]: it's not actually optional.. if you want to do the optional part is is rather required to have those two installed 😄

Comment (kuceriklukas) [reply image-service-grpc/README.md:49]: I'll rephrase it to "required for the optional part"? 😄
